### DISCUSSION TITLE
Problemi con Meross plugs ed eventuale passaggio a MQTT

### DISCUSSION THREAD

**Discussion by Topolynx** [indomus/forum#260] [Hardware & Firmware]
Ciao a tutti. Ho di nuovo bisogno di un vostro consiglio. Sono reduce da un ennesimo impazzimento di due su tre smart plugs della Meross, gli mss310r. Praticamente si perdono la connessione Wi-Fi (con una potenza di segnale tra il 95 e il 100% eh) e diventano irraggiungibili anche per due giorni di fila, riassestandosi infine da sé. Il problema parrebbe a monte di HA (in cui sono integrate via HACS con il plugin di Alberto Geniola), a meno che HA non interferisca con il polling dei server Meross.
Anzitutto volevo sapere se qualcun altro con le mie stesse prese ha riscontrato questo problema, e poi volevo sapere se sia il caso di metter su il mio MQTT broker sul raspberry tentando eventualmente di tagliare i ponti con i server Meross.

**Comment by Scialla** [comment]
quando succede le raggiungi con la loro app?
Se no, il problema è la tua rete wifi; quanti dispositivi collegati hai? alcuni router hanno limiti sul numero di dispositivi collegabili contemporaneamente alla wifi.
Tagliare i ponti con il cloud meross e possibile ma non semplice, Geniola stava cercando di creare qualcosa in tal senso ma non ho più seguito e non so se sia riuscito o meno

**Reply by Topolynx** [reply]
No no! Quando succede ovviamente anche e soprattutto l’app non va più. Questo problema lo hanno sempre dato, tanto che poi sono passato a prese Tuya che questo lavoro non lo fanno. In totale adesso ho una decina di dispositivi ma anche un router in grado di gestire questi collegamenti.
